sui_core/src/authority.rs

```diff
@@ -181,11 +183,26 @@ pub struct AuthorityState {
     // Fixed size, static, identity of the authority
     /// The name of this authority.
     pub name: AuthorityName,
-    /// Committee of this Sui instance.
-    pub committee: Committee,
     /// The signature key of the authority.
     pub secret: StableSyncAuthoritySigner,
 
+    /// Committee of this Sui instance.
+    pub committee: Committee,
+    /// Sequence number of the first checkpoint of the current epoch.
+    /// This will only be set when an authority restarts with new epoch information.
+    /// Hence there will never be data races.
+    #[allow(dead_code)]
+    first_checkpoint: CheckpointSequenceNumber,
```

Comment:
Is the first checkpoint of an epoch not always 0?

Reply:
Do we prefer to reset checkpoint number at each epoch? I was thinking just letting checkpoint number grow naturally.